examples/src/main/java/io/dapr/examples/secrets/README.md

```diff
@@ -46,167 +43,93 @@ Then get into the examples directory:
-export VAULT_ADDR=http://127.0.0.1:8200/
-```
-
-Login to Hashicorp's Vault:
-
-<!-- STEP
-name: Vault login
-expected_stdout_lines:
-  - "Success! You are now authenticated. The token information displayed below"
-  - "token                myroot"
-env:
-  VAULT_ADDR: "http://127.0.0.1:8200/"
--->
-
-```bash
-vault login myroot
+echo '{"redisPassword":"root123","randomKey":"value"}' > ./components/secrets/secret.json
 ```
 
 <!-- END_STEP -->
 
-Create secret (replace `$MY_FAVORITE_MOVIE` with a title of our choice):
-
-<!-- STEP
-name: Create movie vault secret
-expected_stdout_lines:
-  - "version            1"
-env:
-  VAULT_ADDR: "http://127.0.0.1:8200/"
-  MY_FAVORITE_MOVIE: "Star Wars"
--->
-
-```bash
-vault kv put secret/dapr/movie title="$MY_FAVORITE_MOVIE"
-```
-
-<!-- END_STEP -->
-
-Create random secret:
-
-<!-- STEP
-name: Create random vault secret
-expected_stdout_lines:
-  - "version            1"
-env:
-  VAULT_ADDR: "http://127.0.0.1:8200/"
--->
-
-```bash
-vault kv put secret/dapr/randomKey testVal="value"
-```
-
-<!-- END_STEP -->
-
-In the command above, `secret` means the secret engine in Hashicorp's Vault.
-Then, `dapr` is the prefix as defined in `< repo dir >/examples/components/hashicorp_vault.yaml`.
-Finally, `movie` and `randomKey` are the secret names with the value set in the form of `key=value` pair.
-
-A secret in Dapr is a dictionary.
-
 ### Running the secret store sample
 
 The example's main function is in `SecretClient.java`.
 
 ```java
 public class SecretClient {
+
     /**
      * Identifier in Dapr for the secret store.
      */
-    private static final String SECRET_STORE_NAME = "vault";
-  
+    private static final String SECRET_STORE_NAME = "local-secret-store";
+
     /**
      * JSON Serializer to print output.
      */
     private static final ObjectMapper JSON_SERIALIZER = new ObjectMapper();
-  
-  ///...
-
-  public static void main(String[] args) throws Exception {
-      if (args.length != 1) {
-        throw new IllegalArgumentException("Use one argument: secret's key to be retrieved.");
-      }
-  
-      String secretKey = args[0];
-      try (DaprClient client = (new DaprClientBuilder()).build()) {
-        Map<String, String> secret = client.getSecret(SECRET_STORE_NAME, secretKey).block();
-        System.out.println(JSON_SERIALIZER.writeValueAsString(secret));
-
-        try {
-          secret = client.getSecret(SECRET_STORE_NAME, "randomKey").block();
-          System.out.println(JSON_SERIALIZER.writeValueAsString(secret));
+
+    /**
+     * Client to read a secret.
+     *
+     * @param args Unused arguments.
+     */
+    public static void main(String[] args) throws Exception {
+        if (args.length != 2) {
+            throw new IllegalArgumentException("Use two argument: secret's key to be retrieved.");
+        }
+
+        String firstSecretKey = args[0];
+        String secondSecretKey = args[1];
+        try (DaprClient client = (new DaprClientBuilder()).build()) {
```

Comment:
Instead, change the example to accept multiple arguments, not only two. Basically, making this a loop.

Reply:
I think. We can accept arguments that user input on the command line using the api like `Scanner` perhaps. So the example can accept any store-name and any secret key to retrieve. but run this example with command `dapr run --components-path ./components/secrets --app-id secrets1 -- java -jar target/dapr-java-sdk-examples-exec.jar io.dapr.examples.secrets.SecretClient` can't be accept argument from command and throw  `No line found exception`.
So , user need to open two terminal that one run dapr process and other run example process. Hope to get your advice